### PR TITLE
Feature/open leaderboard

### PR DESCRIPTION
**Pull request by polsevev**
This is kinda cursed, but its a quick fix and we should probably revert it when it is time for us to host our own :)

### CODE REVIEW

**Review by skivdal — APPROVED** [review]
Agree that its kinda cursed, but the code looks good

**Review by simsine — APPROVED** [review]

**Review by Celsiusss — APPROVED** [review]

File: friByte.capture-the-flag.service/friByte.capture-the-flag.service/Controllers/LeaderboardController.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace friByte.capture_the_flag.service.Controllers;
 /// <summary>
 /// API endpoints for serving leaderboard
 /// </summary>
-[Authorize(Roles = $"{IdentityRoleNames.TeamRoleName}, {IdentityRoleNames.AdminRoleName}")]
+//[Authorize(Roles = $"{IdentityRoleNames.TeamRoleName}, {IdentityRoleNames.AdminRoleName}")]
```
Comment (mathiash98) [inline friByte.capture-the-flag.service/friByte.capture-the-flag.service/Controllers/LeaderboardController.cs:11]: I think it's perfectly valid to keep this public forever

**Review by mathiash98 — APPROVED** [review]
Made a few changes to make it less of a hack. I think it's perfectly valid to keep leaderboard public 👍🏼

File: friByte.capture-the-flag.service/friByte.capture-the-flag.service/Controllers/LeaderboardController.cs (hunk 1 above)
Comment (polsevev) [reply friByte.capture-the-flag.service/friByte.capture-the-flag.service/Controllers/LeaderboardController.cs:11]: Alrighty!

**Pull request merged**